.vscode/settings.json

```diff
@@ -4,5 +4,8 @@
   "[dockerfile]": {
     "editor.defaultFormatter": "ms-azuretools.vscode-docker"
   },
-  "cSpell.words": ["bigfood", "codepaths", "nestjs", "preact", "tsbuildinfo"]
+  "cSpell.words": ["bigfood", "codepaths", "nestjs", "preact", "tsbuildinfo"],
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "ms-vsliveshare.vsliveshare"
```

Comment:
It should be deleted or changed to prettier.

Reply:
vscode file added to gitignore.

Reply:
`.vscode/settings.json` should be uploaded to git. Because this file indicates settings that should be applied to collaborators in common.
But the 8-9 line is invalid. We don't use `ms-vsliveshare.vsliveshare` as default formatter.
Let's just remove 8-9 line.

Reply:
removed 8-9 line.